android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java

```diff
@@ -594,4 +595,11 @@ public String getApiName()
 	{
 		return "Ti.UI.Window";
 	}
+
+	private void logDeprecatedPropertiesWarning() {
```

Comment:
I don't understand?

```Java
import org.appcelerator.titanium.TiC;
```

Allows you to access any constant in TiC, regardless if it's used in the method or not.